lib/puppet_x/templates/init_ps.ps1

```diff
@@ -207,6 +215,17 @@ namespace Puppet
       }
     }
 
+    public string StdErr
```

Comment:
Because `Output` is not the same thing.  `Output` is the result of collecting all the calls from the PowerShell `Write-XXX` style cmdlets.

In this case, we're actually redirecting `[Console]::Error` writes, which is a proxy for `stderr` (PowerShell makes all the various stream sink concepts a bit confusing TBH)